hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/mongodb/MongoSqlConnectorTest.java

```diff
@@ -176,8 +176,9 @@ public void readWithOneProcessor() {
         execute("CREATE MAPPING " + collectionName + " (firstName VARCHAR, lastName VARCHAR, jedi BOOLEAN) "
                 + "DATA CONNECTION testMongo "
                 + "OPTIONS ("
-                + "    'forceMongoReadParallelismOne' = 'true' "
-                + ")");
+                + "    'forceReadParallelismOne' = 'true' "
```

Comment:
breaking change?

Reply:
Mongo was in BETA, in 5.4 it's planned to go GA.